src/test/java/com/google/cloud/teleport/templates/PubsubToBigQueryTest.java

```diff
@@ -16,29 +16,42 @@
 
 package com.google.cloud.teleport.templates;
 
+import static com.google.cloud.teleport.templates.TextToBigQueryStreaming.wrapBigQueryInsertError;
```

Comment:
Do you want to instead add some unit tests for the TextToBigQueryStreaming pipeline instead? i.e. Add TextToBigQueryStreamingTest ?

Reply:
okay, i try add the unit tests for TextToBigQueryStreamingTest.

Don't you need a E2E test of the error be caused?
I remove that, but I will return if needed.